use-the-built-in-sql-client.md

```diff
@@ -18,6 +18,7 @@ $ ./cockroach sql <flags>
 # Execute SQL from the command line:
 $ ./cockroach sql --execute="<sql statement>;<sql statement>" --execute="<sql-statement>" <flags>
 $ echo "<sql statement>;<sql statement>" | ./cockroach sql <flags>
+$ ./cockroach sql <flags> < file-containing-statements.sql
```

Comment:
Yeah, I agree this is a bit confusing, but I'd rather not go against the <flags> convention used in every other doc.